server/src/main/java/org/elasticsearch/cluster/metadata/MetadataIndexTemplateService.java

```diff
@@ -385,16 +400,27 @@ ClusterState addIndexTemplateV2(final ClusterState currentState, final boolean c
      */
     static Map<String, List<String>> findConflictingV2Templates(final ClusterState state, final String candidateName,
```

Comment:
do we still need this? would we ever want to find the conflicting templates without checking the priority?

Reply:
This is used when adding V1 templates, when any overlapping V2 template raises warning, despite its priority